Tools/ssl/multissltests.py

```diff
@@ -1,12 +1,12 @@
 #!./python
-"""Run Python tests against multiple installations of OpenSSL and LibreSSL
```

Comment:
Please leave this test helper unmodified. The LibreSSL code will be useful as soon as LibreSSL becomes API compatible with OpenSSL 1.1.1.